### ISSUE TITLE
Failed to run Redshift

### ISSUE THREAD

**Issue by ishankhare07** [jonls/redshift#301]
![screenshot from 2016-01-16 02-23-54](https://cloud.githubusercontent.com/assets/7482025/12365026/3672f214-bbf8-11e5-94c4-3306d96208a6.png)

unable to connect to geoclue

**Comment by maandree** [comment]
GeoClue does not always run out of the box. You can specify your location manually in the configuration file.

**Comment by ishankhare07** [comment]
@maandree how do to do that?
where can i find this configuration file?
is there a command-line option as well?

**Comment by maandree** [comment]
`~/.config/redshift.conf`
```
[redshift]
location-provider=manual
[manual]
lat=LATITUDE
lon=LONGITUDE
```

or `redshift -l LATITUDE:LONGITUDE`.

**Comment by jonls** [comment]
This may be solved with #500. Please reopen if issues persist in next release.
